auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneNumberVerificationHandler.java

```diff
@@ -27,22 +28,21 @@ public PhoneNumberVerificationHandler(Application application) {
         super(application);
     }
 
-    public void verifyPhoneNumber(@NonNull Activity activity, final String number, boolean force) {
+    public void verifyPhoneNumber(Activity activity, final String number, boolean force) {
```

Comment:
Seems like I removed the annotation when merging the branches

```suggestion
    public void verifyPhoneNumber(@NonNull Activity activity, final String number, boolean force) {
```